app/controllers/admin/snap_applications_controller.rb

```diff
@@ -17,5 +17,25 @@ class SnapApplicationsController < Admin::ApplicationController
 
     # See https://administrate-prototype.herokuapp.com/customizing_controller_actions
     # for more information
+    #
+    def resend_fax
+      application = SnapApplication.find(params[:id])
+      application.exports.faxed.succeeded.update_all(
+        status: :failed,
+        metadata: "Reset by admin",
+      )
+      Export.create_and_enqueue!(snap_application: application, destination:
+                                 :fax)
+      flash[:notice] = "Resent fax #{application.email}!"
+      redirect_to admin_root_path
+    end
+
+    def pdf
+      application = SnapApplication.find(params[:id])
+      send_data(application.pdf.read,
+        filename: "snap-application-#{application.email}.pdf",
```

Comment:
email is not a required attribute -- maybe use DOB or name instead?

Reply:
Good call. Using signature.